src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -354,6 +355,15 @@
   {% endspaceless %}
 {% endblock date_range_widget %}
 
+{% block color_picker_widget %}
```

Comment:
I don't know if this should be added in this file, with the work that @eternoendless to uniformise the ui kit form theme, if this block is added here it won't be available in the new form theme
@eternoendless what would you recommend?

Reply:
@eternoendless recommended to put it the UI KIt 
https://github.com/PrestaShop/PrestaShop/pull/17215#discussion_r373081131
As I don't know how it works yet, I will do it later